internal/driver/nodeserver.go

```diff
@@ -236,53 +234,16 @@ func (ns *LinodeNodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeSt
 		return &csi.NodeStageVolumeResponse{}, nil
 	}
 
-	// VolumeMode=block
+	// Check if the volume mode is set to 'Block'
 	// Do nothing else with the mount point for stage
-	if blk := volumeCapability.GetBlock(); blk != nil {
+	if blk := req.VolumeCapability.GetBlock(); blk != nil {
```

Comment:
Do we need a `nil` check on `req.VolumeCapability` here since the original implementation had one?

Reply:
Yup. I'm trying to keep the original code the same and not make any changes unless absolutely needed

Reply:
Oh, I was not referring to the return value of `GetBlock`, but `req.VolumeCapability` itself. I just noticed that there was a nil check on it before but I don't see it being done here.